performant-software/Annotation-Studio
# Add loading indicator for document show

## What does this PR do?
1. Add a loading indicator when the user opens up any document on the document show page

## How to test?
* Go to http://cove-staging.herokuapp.com/documents and click on any document (Eg. http://cove-staging.herokuapp.com/documents/goblin-market
* Confirm there is a loading circle there until the document is fully loaded
